spec/datadog/tracing_spec.rb

```diff
@@ -0,0 +1,170 @@
+# rubocop:disable RSpec/VerifiedDoubles
+# All the doubles in this file are simple pass through values.
+# There's no value in making them verifying doubles.
+RSpec.describe Datadog::Tracing do
+  describe '.active_span' do
+    subject(:active_span) { described_class.active_span }
+
+    it 'delegates to the tracer' do
+      expect(Datadog.tracer).to receive(:active_span)
+      active_span
+    end
```

Comment:
Minor: Just for a bit of extra coverage, I guess we could validate that the result of `active_span` is the result of `Datadog.tracer.active_span`.

Of course, right now the method is trivial, but since it's a public API, having a test will keep us in check if we decide to change it :)

Reply:
I've added this coverage to all methods that we care about the return value here.